java-symbol-solver-core/src/test/java/com/github/javaparser/symbolsolver/model/typesystem/TypeVariableUsageTest.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2016 Federico Tomassetti
```

Comment:
I've just looked at the Apache License again and it seems that we would produce an infinite recursion if we'd really change it to "Contributors of JavaSymbolSolver" ;) The license states:
```
"Licensor" shall mean the copyright owner or entity authorized by
the copyright owner that is granting the License.
```
So maybe the "JavaSymbolSolver team" is the `Licensor` authorized by the `copyright owner`, but I think it's best if we keep @ftomassetti as the `copyright owner`. Otherwise the license becomes hard to understand.